README.md

```diff
@@ -89,7 +89,7 @@ The following example shows how the GraphQL API Composer can be used with Fastif
 'use strict';
 const { compose } = require('@platformatic/graphql-composer')
 const Fastify = require('fastify')
-const { buildClientSchema } = require('graphql')
+const { printSchema, buildClientSchema } = require('graphql')
```

Comment:
It might be worth updating https://github.com/platformatic/graphql-composer/blob/bde7b29c43f205152b81fe87537eb22f83d23794/test/helper.js#L75 as well just to make sure the docs and tests are in sync and working.

Reply:
How about add a `.toSdl` method?

Reply:
That sounds good to me.